src/responsive-layout.tsx

```diff
@@ -41,6 +42,9 @@ const useStyles = createUseStyles((theme) => ({
     },
 }));
 
+const ResponsiveLayoutMarginContext = React.createContext<null | number>(0);
+export const useResonsiveLayoutMargin = (): null | number => React.useContext(ResponsiveLayoutMarginContext);
+
```

Comment:
if you use <ResponsiveLayout fullWidth> with carousel, in desktop arrows positions are out of the browser
https://tinyurl.com/y9snav2v

Reply:
I think it is not very important by the moment, but it is a bug anyway

Reply:
don't use fullwidth prop in `ResponsiveLayout`. I think we should review and deprecate that prop